src/main/views/courts/tabs/casesHeardContent.njk

```diff
@@ -0,0 +1,81 @@
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+<h2 class="govuk-heading-l">Cases Heard</h2>
+
+{% if errors|length %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: errors
+  }) }}
+{% endif %}
+
+{% if updated === true %}
+  {{ govukPanel({
+    titleText: "Cases heard updated"
+  }) }}
+{% endif %}
+
+{% if allAreasOfLaw|length %}
+
+  {% set courtAreasOfLawNames = [] %}
+  {% for areaOfLaw in courtAreasOfLaw %}
+    {% set allAreasOfLawItems =
+      (courtAreasOfLawNames.push(areaOfLaw.name), courtAreasOfLawNames) %}
+  {% endfor %}
+
+  {% set allAreasOfLawItems = [] %}
```

Comment:
allAreasOfLawItems is being set to an empty array here but was populated in a loop just before this in lines 27-28.